.github/workflows/assign-issue-owners.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
 
       - name: Install js-yaml
-        run: npm install js-yaml
+        run: npm install js-yaml@4.0.0
```

Comment:
intentionally using version prior to latest so we can see right away if renovate custom manager is working